tests/rbe_repo/BUILD

```diff
@@ -278,10 +278,13 @@ file_test(
 
 # Tests validate run_and_extract uses mount volumes (-v) vs
 # data volumes.
+# Set with tag 'manual' as this test messes up others if the
```

Comment:
Could you briefly mention here what aspect of the test results in this issue?

Reply:
added more details. ptal